cmd/neofs-ir/defaults.go

```diff
@@ -32,13 +32,9 @@ func newConfig(path string) (*viper.Viper, error) {
 func defaultConfiguration(cfg *viper.Viper) {
 	cfg.SetDefault("logger.level", "info")
 
-	cfg.SetDefault("pprof.enabled", false)
-	cfg.SetDefault("pprof.address", ":6060")
-	cfg.SetDefault("pprof.shutdown_ttl", "30s")
+	cfg.SetDefault("profiler.shutdown_timeout", "30s")
```

Comment:
Please add `cfg.SetDefault("profiler.address", "")` and `cfg.SetDefault("metrics.address", "")` for inner ring app.

We discussed it already in the context of alphabet contracts defaults. However in this case there are only two meaningful config records. I want to keep all available configuration values in one place as mush as it possible.

Reply:
Done.